File: server/src/main/java/com/upsintern/staffsync/dto/EventDto.java
package com.upsintern.staffsync.dto;


import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class EventDto {
    String eventName;
    String eventVenue;
    Date eventDate;
    String eventDescription;
}

File: client/src/app/model/event-dto.ts
export class EventDto {
  eventName:String;
  eventVenue:String;
  eventDate:String;
  eventDescription:String;

  constructor(eventName: String, eventVenue: String, eventDate: String, eventDescription: String) {
    this.eventName = eventName;
    this.eventVenue = eventVenue;
    this.eventDate = eventDate;
    this.eventDescription = eventDescription;
  }

}
